1.7/administration/storage/nfs.md

```diff
@@ -5,14 +5,14 @@ menu_order: 1
 
 # Overview
 
-For some stateful services such as Jenkins/Velocity, it can be convenient to mount a shared network drive to every node. This facilitates launching the task on a new node if the node in use becomes unavailable.
+For some stateful services, such as Jenkins/Velocity, it can be convenient to mount a shared network drive to every node. A shared network drive makes it possible to launch the task on a new node if the node in use becomes unavailable.
 
-This example uses CoreOS and systemd and has not been tested in other environments.
+**Note:** This example uses CoreOS and systemd and has not been tested in other environments.
 
 ## Notes
 
-- These instructions are CoreOS specific.
-- This is not an HA NFS server. It is being run on a single master and should only be used as a proof of concept.
+- These instructions are CoreOS-specific.
+- This is not an HA NFS server. It runs on a single master and should only be used as a proof of concept.
```

Comment:
Would also like to add that most operations teams will have access to NFS or another network store, and those folks can skip right to Part 2.